main/inc/lib/message.lib.php

```diff
@@ -1472,13 +1493,25 @@ public static function showMessageBox($messageId, $type)
                     Display::return_icon('delete.png', get_lang('DeleteMessage')).'</a>&nbsp';
                 break;
             case self::MESSAGE_TYPE_INBOX:
-                $message_content .= '<a href="inbox.php?'.$social_link.'">'.
-                    Display::return_icon('back.png', get_lang('ReturnToInbox')).'</a> &nbsp';
-                $message_content .= '<a href="new_message.php?re_id='.$messageId.'&'.$social_link.'">'.
-                    Display::return_icon('message_reply.png', get_lang('ReplyToMessage')).'</a> &nbsp';
-                $message_content .= '<a href="inbox.php?action=deleteone&id='.$messageId.'&'.$social_link.'" >'.
-                    Display::return_icon('delete.png', get_lang('DeleteMessage')).'</a>&nbsp';
-                break;
+                    if ($idNextMessage != 0)
+                        {
+                            $message_content .= '<a title="' . get_lang('ScormNext') . '" href="view_message.php?type=' . $type . '&id=' . $idNextMessage . '" style="float:right;">' .
```

Comment:
Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 183 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)